modal/sandbox.py

```diff
@@ -91,7 +91,36 @@ async def _get_logs():
         return data
 
 
+class _StreamWriter:
```

Comment:
I call it the `_StreamWriter` to mirror `asyncio.subprocess`'s `StreamWriter`. Happy to rename to `LogWriter` or something else.

Reply:
Called out IRL that the write methods should actually be sync if we're following the asyncio interface

Reply:
good call!